docs/getting_started/folders_and_files/index.md

```diff
@@ -1 +1,3 @@
-A BIDS dataset basically consists of a collection of folders and files.
+# BIDS Folders and Files
+
+A BIDS dataset consists of a collection of folders and files that are organized and named in compliance with BIDS data outlines. The following sections provide more information on how to properly organize and name your data. 
```

Comment:
@iamdamion let's find a better way to say this, and/or maybe clarify with a link.

Reply:
updated